fe/fe-core/pom.xml

```diff
@@ -317,24 +316,6 @@ under the License.
             </exclusions>
         </dependency>
 
-        <dependency>
-            <groupId>software.amazon.awssdk</groupId>
-            <artifactId>bundle</artifactId>
-            <version>${awsv2.version}</version>
```

Comment:
exclude unused dependency

Reply:
hard do it, hadoop-aws will include `bundle` automatically.